src/dmd/globals.d

```diff
@@ -198,6 +198,10 @@ struct Param
     const(char)* jsonfilename;          // write JSON file to jsonfilename
     JsonFieldFlags jsonFieldFlags;      // JSON field flags to include
 
+    OutBuffer* mixinOut;                // write expanded mixins for debugging
+    const(char)* mixinFile;             // .mixin file output name
+    int mixinLines;                     // Number of lines in writeMixins
+
```

Comment:
mixinOut and mixinLines should better go into the Global struct rather than Param. I guess you followed the moduleDeps example, but I'd consider it a bad one.